test/tests/performance/pbs_sched_perf.py

```diff
@@ -164,3 +165,53 @@ def test_node_bucket_perf_free(self):
         """
         num_jobs = 3000
         self.compare_normal_path_to_buckets('free', num_jobs)
+
+    @timeout(3600)
+    def test_pset_fuzzy_perf(self):
```

Comment:
can you also write a similar PTL test where the number of placement sets is larger, and opt_backfill_fuzzy is set to off, to just make sure that the code changes have not slowed the scheduler down for such a configuration?

Reply:
I don't think that test is worth writing, but as it is, I crafted this test to give us that result.  If opt_backfill_fuzzy is off, we stop and check after every event.  By default we stop and check after every minute.  The jobs in this test have an increasing walltime of 1m each.  This means we'll stop and check after every event.